docs/Plugins/webhook.md

```diff
@@ -220,7 +228,7 @@ workflows:
 
 
 
-Actually, we finish the webhook in prev step. If we want to do more, the config in CircleCi is as fellow. It will call webhook before tasks start and after tasks fail.
+Actually, the incoming webhook works in prev step. If you want to do more, the config in CircleCI is as fellow. It will call webhook before tasks start and after tasks fail.
```

Comment:
I'm not understanding the first sentence: (Actually, the incoming webhook works in prev step). The original version made more sense.
As for the rest, replace with:

If you want to customize more, a sample CircleCI config is provided below. This sample calls the webhook before the tasks start and after tasks fail.